src/screens/Forum/ForumPost.js

```diff
@@ -68,8 +68,41 @@ export default function ForumPost({
   const handleEdit = () => {
     // TODO: add ability to edit post if belongsToCurrentUser
   };
+
   const handleDelete = () => {
-    // TODO: delete this post if belongsToCurrentUser
+    // Pop up confirmation
+    Alert.alert(
```

Comment:
Like I said during the live code review, try using the material UI dialog box instead of the react native alert